src/schemas/raster.schema.json

```diff
@@ -49,11 +49,12 @@
     "metadata": {
       "type": "object",
       "additionalProperties": false,
-      "anyOf":[{"required": ["dimensions","isPyramid"]},{"required": ["transform"]},{"required": ["omeTiffOffsetsUrl"]}],
+      "anyOf":[{"required": ["dimensions","isPyramid"]},{"required": ["transform"]},{"required": ["omeTiffOffsetsUrl"]},{"required": ["isBitmask"]}],
```

Comment:
Do you think the boolean will be future-proof or should it be a string / categorical value instead? Do we anticipate other types of layers in the future (other than bitmask or raster)?

Reply:
Hmmm, we could just make this `type` which would be one-to-one with the layer where it's either `layer.type = 'bitmask'` or `layer.type = 'raster'`.  I can't think of any other raster-type layers though

Reply:
Ok we can keep the boolean, I think having two `type` fields would be confusing